stream.js

```diff
@@ -43,18 +43,28 @@ function WebSocketStream(target, protocols, options) {
   }
 
   // was already open when passed in
-  if (socket.readyState === 1) {
+  if (socket.readyState === WS.OPEN) {
```

Comment:
I've checked it's part of the 'ws' library and the browsers too. But I can revert it if you would like to.